plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -229,11 +234,16 @@ public IcebergSessionProperties(
                         PARQUET_WRITER_BLOCK_SIZE,
                         "Parquet: Writer block size",
                         parquetWriterConfig.getBlockSize(),
+                        value -> validateMaxDataSize(PARQUET_WRITER_BLOCK_SIZE, value, DataSize.valueOf(PARQUET_WRITER_MAX_BLOCK_SIZE)),
```

Comment:
Note: I had to drop the PARQUET_WRITER_MIN_BLOCK_SIZE validation here to allow certain tests in iceberg which rely on producing small row groups to pass.

Reply:
that means PARQUET_WRITER_MIN_BLOCK_SIZE is not a real min size.
is it safe not to have min-value for writer block size?
if safe, we should remove PARQUET_WRITER_MIN_BLOCK_SIZE altogether

Reply:
It is safe to provide a low value for row group size, just not recommended for good performance. Most users will be using this through config properties rather than session property. But I don't have strong preference here, I am fine with removing PARQUET_WRITER_MIN_BLOCK_SIZE or keeping it in current shape.

Reply:
it seems kind of weird to have PARQUET_WRITER_MIN_BLOCK_SIZE and enforce it for Hive, Delta, but not Hive.
is this what we really want?

Reply:
If inconsistency between connectors is the main concern, I can drop PARQUET_WRITER_MIN_BLOCK_SIZE session validation from all 3 connectors.
If inconsistency between session and config validation is also a problem, then I will drop PARQUET_WRITER_MIN_BLOCK_SIZE entirely. Let me know which is preferable.

Reply:
idk what is this for, so don't want to suggest particular route, but let's not leave this in such a state.

Reply:
This is for enforcing a reasonable minimum size for parquet row groups. Using very small row groups is bad for performance. This validation is a nice to have, not strictly necessary to enforce as so far all of these have been unvalidated properties and using a small number will also work.

Reply:
ok, so if we can leave without validation, let's remove it.
if we want validation, let's not omit Iceberg. I think Iceberg is an important connector

Reply:
Okay I'll remove PARQUET_WRITER_MIN_BLOCK_SIZE in another PR

Reply:
https://github.com/trinodb/trino/pull/17625